Applications/DataExplorer/DataView/NetCdfConfigureDialog.cpp

```diff
@@ -43,7 +41,7 @@ NetCdfConfigureDialog::~NetCdfConfigureDialog()
 void NetCdfConfigureDialog::accept()
 {
 	QMessageBox valueErrorBox;
-	if (_currentVar->num_dims() < 3){
+	if (_currentVar->num_dims() < 2){
```

Comment:
Just out of curiosity: What is `_currentVar` used for and why a return value of `num_dim()` smaller than 2 not sufficient? A suggestion: if you invested some time to understand the class and to solve the issues it would be very nice if you could add some documentation to the class. I hope you will find the time! :smile: